Add search bar

CathrinTruchan opened the pull request:
Hello :) 
I added a search bar to the index and the flow-page. I am looking forward to your feedback. Thanks :)

F-Kirchhoff reviewed (changes requested):
Hey, I added some suggestions, I think you made it yourself much harder than it needed to be

CathrinTruchan commented on pages/_app.js:
```diff
@@ -2,10 +2,16 @@ import GlobalStyle from "../components/GlobalStyle";
 
 function MyApp({ Component, pageProps }) {
+  const [searchQuery, setSearchQuery] = useState("");
+
   return (
     <>
       <GlobalStyle />
-      <Component {...pageProps} />
+      <Component
+        {...pageProps}
+        searchQuery={searchQuery}
```
Ah, of course. That's easier :) Thanks!

CathrinTruchan commented:
> Hey, I added some suggestions, I think you made it yourself much harder than it needed to be

Thanks a lot for reviewing @F-Kirchhoff! I added the changes you requested. Can you check the code again?

F-Kirchhoff commented on pages/index.js:
```diff
@@ -17,10 +19,13 @@ export default function Home({ searchQuery, setSearchQuery }) {
-            const searchQueryInLowerCase = searchQuery.toLowerCase();
-            if (nameInLowerCase.includes(searchQueryInLowerCase)) {
+          {asanas
+            .filter((asana) => {
+              const nameInLowerCase = asana.english_name.toLowerCase();
+              const searchQueryInLowerCase = searchQuery.toLowerCase();
+              return nameInLowerCase.includes(searchQueryInLowerCase);
+            })
```
nice!

F-Kirchhoff reviewed (approved):
LGTM 🦊

CathrinTruchan commented:
> LGTM 🦊

Thanks ✌️
